tests/unit/test_utils_types_containers.py

```
@@ -0,0 +1 @@
+from lighter.utils.types.containers import nested
```

Ruff (0.8.2)

`lighter.utils.types.containers.nested` imported but unused

Remove unused import: `lighter.utils.types.containers.nested`

(F401)